.github/workflows/post-deploy.yml

```diff
@@ -22,15 +22,19 @@ on:
         required: false
         default: false
         type: boolean
+      app_name:
+        required: true
+        default: false
+        type: string
 
-    # not required when not deploying to cf
     secrets:
       GOV_NOTIFY_API_KEY:
```

Comment:
This needs deleting, not just the comment...

Reply:
But then it doesn't work.

Reply:
Plus Gideon did tell me one of the issues I had before was not having GOV_NOTIFY_API_KEY.